server/engine/src/test/resources/cfast/case_execCicsReturn.result.json

```diff
@@ -4,19 +4,14 @@
     "location": {
       "uri": "fake/path",
       "start": {
-        "line": 3,
-        "character": 9
+        "line": 3.0,
```

Comment:
There is no use in having float types for line/character. I suggest to revert that file and (re?) adjust the "origin" to use plain "int" for those attributes instead (there's no use in a full `Integer` object either).

Reply:
updated

Reply:
The open question is: Where did the float values came from in the first place.
Isn't there some place that uses a floating type that _should_ use plain `int`?

Reply:
The origin has it as int (org.eclipse.lsp4j.Position). 
To assert we use [GSON.toJson](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/403ef773f601db0c6639a5b633aae2007f5b7c49/server/engine/src/test/java/org/eclipse/lsp/cobol/extendedapi/CFASTBuilderTest.java#L63) , which converts int to float (default behavior).

Also, I simply checked the structure of the JSON and didn't care for the float (my bad)

Reply:
OK - while I see https://javadoc.io/doc/org.eclipse.lsp4j/org.eclipse.lsp4j/latest/org/eclipse/lsp4j/Position.html, where is `org.eclipse.lsp4j.Position` actually defined (it seems to be missing in https://github.com/eclipse-lsp4j/lsp4j/tree/main/org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j)?

I suggest that in another PR `setObjectToNumberStrategy(ToNumberPolicy.LONG_OR_DOUBLE)` may be used https://stackoverflow.com/a/70229243/5027456

Reply:
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2098